src/index.js

```diff
@@ -35,31 +33,26 @@ async function main() {
   datadog.timing('main.init_algolia', Date.now() - start);
 
   // Create State Manager that holds progression of indexing
-  stateManager = createStateManager(mainIndex);
+  const stateManager = createStateManager(mainIndex);
 
   // then we run the bootstrap
   // after a bootstrap is done, it's moved to main (with settings)
   // if it was already finished, we will set the settings on the main index
   start = Date.now();
   log.info('⛷   Bootstraping');
-  await bootstrap(
-    await stateManager.check(),
-    algoliaClient,
-    mainIndex,
-    bootstrapIndex
-  );
+  await bootstrap(stateManager, algoliaClient, mainIndex, bootstrapIndex);
```

Comment:
seems inconsistent between bootstrap which checks within the function and replicate which checks outside?

Reply:
indeed